website/docs/docs/dbt-cloud/release-notes/09-April-2022/gitlab-auth.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Refreshing Auth in the IDE for GitLab users on the dbt Cloud Enterprise Plan"
+id: "gitlab-auth"
+description: "Adding support for expiring OAuth access tokens."
+sidebar_label: "Enhancement: Refreshing GitLab OAuth Access"
+tags: [May-19-2022, v1.1.52]
+---
+
+GitLab is changing their treatment of [OAuth access tokens that don't expire](https://docs.gitlab.com/ee/update/deprecations.html#oauth-tokens-without-expiration) on May 22. We updated our IDE logic to handle OAuth token expiration more gracefully. Now after 2 hours of consecutive IDE usage, the user will have to re-authenticate in GitLab. This additional security layer in the IDE is available only to the dbt Cloud enterprise tier.
```

Comment:
not sure if this is worth mentioning, but if the user disconnects/reconnects their GitLab account or connects a repo to a project using native GitLab integration, they won't see the modal in the IDE. only one form of "manual" re-auth is necessary to trigger automatic token refresh. otherwise, this looks good to me 🔥